### ISSUE TITLE
Incompatibility with BLAST 2.6.0+

### ISSUE THREAD

**Issue by glucksfall** [paulstothard/cgview_comparison_tool#1]
Hello @paulstothard 

I found a problem trying to run `scripts/process_test_projects.sh`

```
DESCRIPTION
   Protein Query-Translated Subject BLAST 2.6.0+

Use '-help' to print detailed descriptions of command line arguments
========================================================================

Error: Unknown argument: "query_gencode"
Error:  (CArgException::eInvalidArg) Unknown argument: "query_gencode"
Program failed, try executing the command manually.
```

Moreover, I have BLAST 2.9.0 that should be the first option and you included BLAST 2.2.26.

Should I use BLAST 2.2.26 to run your software?

Thank you for your time,
Best regard.

**Comment by paulstothard** [comment]
Hello,

Yes unfortunately the CGView Comparison Tool requires the older "legacy" BLAST, so you will need to use BLAST 2.2.26 or one of the older versions. One some platforms it can be difficult to setup legacy BLAST, so for that reason I often use the CCT Docker container.

Paul

**Comment by glucksfall** [comment]
Thank you for the response.

I will think about a workaround for BLAST because the docker isn't an option to use in our cluster.

Best regard.

**Comment by paulstothard** [comment]
There may be a suitable binary here (the source is provided too):

https://ftp.ncbi.nlm.nih.gov/blast/executables/legacy.NOTSUPPORTED/2.2.26/

**Comment by glucksfall** [comment]
Yes, thank you. The problem is I have installed a default version for BLAST, so the user of your software must configure its environment to work with another BLAST version and not interfere with others.

I modified ```global_settings.conf``` and set the path to blastall and formatdb with the legacy_blast included in your software. I need to inform the final user how to use the configuration file (or create an alias for the software).

Thank you for all your time.
